public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -305,6 +305,9 @@ mciModule.controller('PerformanceDiscoveryCtrl', function(
         field: 'test',
         _link: row => '/task/' + row.entity.taskId + '##' + row.entity.test,
         cellTemplate: 'ui-grid-link',
+        filter: {
+          term: '^((?!canary_.*|fio_.*|iperf.*|NetworkBandwidth).)*$'
```

Comment:
Shouldn't this be  **'^((?!canary_|fio_|iperf|NetworkBandwidth).\*)$'** and it should be a common constant.